simde/x86/avx512/compress.h

```diff
@@ -0,0 +1,40 @@
+#if !defined(SIMDE_X86_AVX512_COMPRESS_H)
+#define SIMDE_X86_AVX512_COMPRESS_H
+
+#include "types.h"
+
+HEDLEY_DIAGNOSTIC_PUSH
+SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
+SIMDE_BEGIN_DECLS_
+
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m256d
+simde_mm256_mask_compress_pd (simde__m256d src, simde__mmask8 k, simde__m256d a) {
+  #if defined(SIMDE_X86_AVX512VL_NATIVE) && defined(SIMDE_X86_AVX512F_NATIVE)
+    return _mm256_mask_compress_pd(src, k, a);
+  #else
+    simde__m256d_private
+    a_ = simde__m256d_to_private(a),
+    src_ = simde__m256d_to_private(src);
+    size_t j = 0;
+    SIMDE_VECTORIZE
+    for (size_t i = 0; i < (sizeof(a_.f64)/sizeof(a_.f64[0])); i++){
+      if((k >> i) & 1) {
+        a_.f64[j++] =  a_.f64[i];
```

Comment:
There is an extra space here after the assignment operator (the =).  There should only be one.